### DISCUSSION TITLE
Announce 2.0 release through "This week in Gnome"

### DISCUSSION THREAD

**Discussion by rien333** [SoongNoonien/mpdevil#99] [General]
I like reading and discovering things via [This Week in Gnome](https://thisweek.gnome.org/posts/2024/03/twig-138/). This blog frequently features applications that have been ported to Adwaita. 

If you're okay with it, I might be able to do a small write up, celebrating and outlining the 2.0 release?

**Comment by SoongNoonien** [comment]
> If you're okay with it, I might be able to do a small write up, celebrating and outlining the 2.0 release?

Yes, sure!
